README.md

```diff
@@ -47,59 +126,27 @@ guias.eoscostarica.io/
 └── README.md
 ```
 
-### Before to start
-
-Somethings you need before getting started
-* git
-* node.js
-
-## Instructions to run the portal locally in Windows 10
-
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. 
-
-Follow the instructions below.
-
-1. Clone this repo
-
-```bash
-$ git clone https://github.com/eoscostarica/guide.eoscostarica.io.git
-``` 
-
-2. Move to the appropriate directory
-
-```bash
-$ cd guide.eoscostarica.io
-```
-
-3. Install all dependencies:
-
-```bash
-$ yarn install
-```
-
-4. Build the project
-
-```bash
-$ yarn build
-```
+## License
 
-5. Run the project:
+MIT © [EOS Costa Rica](https://eoscostarica.io)
 
-```bash
-$ yarn serve
-```
+## Contributing
 
-## About EOS Costa Rica
+We use a Kanban-style board. That's were we prioritize the work. [Go to Project Board](https://github.com/eoscostarica/guide.eoscostarica.io/projects/1).
 
-EOS Costa Rica is an independently-owned, self-funded, bare-metal genesis block producer that provides stable and secure infrastructure for EOSIO blockchains. We support open source software for our community while offering enterprise solutions and custom smart contract development for our clients.
+If you want to make a contribution, please follow the next steps:
 
-[eoscostarica.io](https://eoscostarica.io)
+1. Fork the Project
+2. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
+3. Commit your Changes (`git commit -m '<type>(<scope>): <subject>'`)
```

Comment:
Not sure, could be a great option to add the `git add <file-name>` step, what you guys @steph222 @xavier506 @Edgar097 thinks?

Reply:
> Not sure, could be a great option to add the `git add <file-name>` step, what you guys @steph222 @xavier506 @Edgar097 thinks?

That is already in the gGtHub guide, but I still don't see it wrong to put it there!!

Reply:
maybe a link to the section of the GitHub Guide for more information

Reply:
> 
> 
> maybe a link to the section of the GitHub Guide for more information

Mm no sure about that, remember it is a Readme file, but like I said, not sure, @xavier506 could you give us some feedback about it.

I just added the `git add` step